pennylane/interfaces/execution.py

```diff
@@ -453,6 +457,8 @@ def execute(
             (within :meth:`Device.batch_transform`) to each tape to be executed. The default behaviour
             of the device batch transform is to expand out Hamiltonian measurements into
             constituent terms if not supported on the device.
+        use_device_jacobian_product (Optional[bool]): whether or not to use the device provided jacobian
+            product if it is available.

```

Comment:
@albi3ro my understanding was that the new JVP/VJP methods wouldn't yet be available in the work we are doing for 0.34. Are you intending this PR to go in for 0.34?

Reply:
https://app.shortcut.com/xanaduai/story/39466/the-torch-interface-can-use-the-device-provided-vjp


Yes this is for v0.34.

Device jacobian products will be turned off by default, by opt-in at `qml.execute`.